bin/triton-translate.cpp

```diff
@@ -131,10 +131,11 @@ LogicalResult tritonTranslateMain(int argc, char **argv,
     llvm::outs() << ::triton::translateLLVMIRToPTX(*llvmir, SMArch.getValue(),
                                                    ptxVersion.getValue());
   else if (targetKind == "hsaco") {
+    const std::string arch = "gfx" + GCNArch.getValue();
+    const std::string triple = "amdgcn" + GCNTriple.getValue();
```

Comment:
Default value of GCNTriple is `-amd-amdhsa`, which is not a valid triple.
Adds `amdgcn` to format it properly.